mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/util/InfrastructureParameterBuilder.java

```diff
@@ -193,6 +202,29 @@ public static ParameterDeclaration addExpirationPolicy(ConfigurationDeclaration
     return parameter;
   }
 
+  public static ParameterDeclaration addErrorMappings(OperationDeclaration operation) {
+    ParameterDeclaration parameter = new ParameterDeclaration(ERROR_MAPPINGS_PARAMETER_NAME);
+    parameter.setDescription(ERROR_MAPPINGS_PARAMETER_DESCRIPTION);
+    parameter.setExpressionSupport(NOT_SUPPORTED);
+    parameter.setRequired(false);
+    parameter.setParameterRole(BEHAVIOUR);
+    parameter.setType(new ErrorMappingsTypeBuilder().buildErrorMappingsType(), false);
+    parameter.setLayoutModel(LayoutModel.builder().tabName(ERROR_MAPPING_TAB).build());
+    parameter.setDslConfiguration(ParameterDslConfiguration.builder()
+        .allowsInlineDefinition(true)
+        .allowsReferences(false)
+        .allowTopLevelDefinition(false)
+        .build());
+    parameter.addModelProperty(new QNameModelProperty(MULE_ERROR_MAPPING_QNAME, true));
+    markAsInfrastructure(parameter, 12);
```

Comment:
Check if org.mule.runtime.extension.api.property.SinceMuleVersionModelProperty should be added here.

Reply:
this is a must